crates/sui-graphql-rpc/src/server/version.rs

```diff
@@ -11,7 +12,8 @@ use axum::{
 
 use crate::error::{code, graphql_error_response};
 
-const RPC_VERSION_FULL: &str = env!("CARGO_PKG_VERSION");
+use super::builder::AppState;
+
 const RPC_VERSION_YEAR: &str = env!("CARGO_PKG_VERSION_MAJOR");
 const RPC_VERSION_MONTH: &str = env!("CARGO_PKG_VERSION_MINOR");
```

Comment:
You should probably also get rid of these, and extract the year and month from the `version` field that is passed in, to avoid the two going out of sync.

Reply:
And this may justify making `Version` a more structured type, that separately contains major, minor, patch, and sha fields.

Reply:
(And a `Display` impl that creates the final version field).